origin/src/lib.rs

```diff
@@ -151,23 +148,19 @@ unsafe extern "C" fn rust(mem: *mut usize) -> ! {
     exit(ret)
 }
 
-#[cfg(target_vendor = "mustang")]
 #[repr(transparent)]
 struct SendSyncVoidStar(*mut c_void);
 
-#[cfg(target_vendor = "mustang")]
 unsafe impl Send for SendSyncVoidStar {}
 
-#[cfg(target_vendor = "mustang")]
 unsafe impl Sync for SendSyncVoidStar {}
 
 /// An ABI-conforming `__dso_handle`.
-#[cfg(target_vendor = "mustang")]
+#[link_section = ".mustang.data"]
```

Comment:
Maybe use `.text.mustang` for functions and `.data.mustang` for statics?

Reply:
Ah, good idea. We're currently using `__mustang` as our C-symbol namespace prefix, so I think this comes out as `.text.__mustang`.